baremetal/polarfire-soc-bare-metal-library/src/platform/drivers/mss_qspi/mss_qspi.c

```diff
@@ -1,5 +1,5 @@
 /***************************************************************************//**
- * Copyright 2019-2021 Microchip FPGA Embedded Systems Solutions.
+ * Copyright 2019-2020 Microchip FPGA Embedded Systems Solutions.

```

Comment:
*suggestion* We probably don't want to change copyright statements from other source files.

Reply:
Updated to 2021 to be consistent with 2021.08